cmd/bootstrap/transit/cmd/pull.go

```diff
@@ -0,0 +1,133 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	bootstrap "github.com/onflow/flow-go/cmd/bootstrap/cmd"
+	"github.com/onflow/flow-go/ledger/complete/wal"
+	model "github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	flagToken      string
+	flagNodeRole   string
+	flagBucketName string
+)
+
+// pullCmd represents a command to pull keys and metadata from the Google bucket
+var pullCmd = &cobra.Command{
+	Use:   "pull",
+	Short: "Fetch keys and metadata from the transit server",
+	Long:  `Fetch keys and metadata from the transit server`,
+	Run:   pull,
+}
+
+func init() {
+	rootCmd.AddCommand(pullCmd)
+	addPullCmdFlags()
+}
+
+func addPullCmdFlags() {
+	pullCmd.Flags().StringVar(&flagToken, "token", "", "token provided by the Flow team to access the Transit server")
+	pullCmd.Flags().StringVar(&flagNodeRole, "role", "", `node role (can be "collection", "consensus", "execution", "verification" or "access")`)
+	pullCmd.Flags().StringVar(&flagBucketName, "bucket", "", "the name of the Google Bucket provided by the Flow team")
+
+	_ = pullCmd.MarkFlagRequired("token")
+	_ = pullCmd.MarkFlagRequired("role")
+	_ = pullCmd.MarkFlagRequired("bucket")
+}
+
+// pull keys and metadata from the Google bucket
+func pull(cmd *cobra.Command, args []string) {
+	log.Info().Msg("running pull")
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
```

Comment:
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
```

We can probably give this a bit more time, since it seems like we're downloading the files sequentially.